packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1043,13 +1092,13 @@ class TextureAndroidViewController extends AndroidViewController {
 
   @override
   Future<void> _sendCreateMessage() async {
-    assert(!_size.isEmpty, 'trying to create $TextureAndroidViewController without setting a valid size.');
+    assert(!_initialSize.isEmpty, 'trying to create $TextureAndroidViewController without setting a valid size.');
```

Comment:
Is `_initialSize` used after creation? Is there a way to refactor the code (without breaking change) so that we don't need this ivar?

Reply:
No. `_size` had the same issue too.  This can be cleaned up if `_sendCreateMessage` receives the size from the `create` call.  However, this will need to be an optional parameter to prevent breaking changes. e.g. https://github.com/flutter/plugins/blob/7686be7ecbe1cc049423715b5e2d897cb63a43c1/packages/webview_flutter/webview_flutter_android/lib/webview_surface_android.dart#L73

I'm inclined to not do that in this PR